automation/src/test/scala/org/broadinstitute/dsde/firecloud/fixture/DownloadFixtures.scala

```diff
@@ -0,0 +1,96 @@
+package org.broadinstitute.dsde.firecloud.fixture
```

Comment:
Added a test - the code to download from a link was in WorkspaceDataPage so I broke it out into `DownloadFixtures`.